README.md

```diff
@@ -1,26 +1,94 @@
-spiped
-------
+spiped and spipe
```

Comment:
I refer to the entire package as `spiped`, even though it now contains two utilities.  In the future there might be even more utilities, e.g., if asymmetric keying is added, but the package as a whole will still be "spiped".